### PR TITLE
Assignment 6 - Group 09

### PR DESCRIPTION
**Pull request by agouloumis**

### CODE REVIEW

**Comment by agouloumis** [comment]
Only take into account the last commits for assignment 6. The other commits are already in another pull request named "Group 09 Assignment 5", in https://github.com/FacultadInformatica-LinkedData/Curso2020-2021-ODKG/pull/257

Sorry for the inconvenience.

File: HandsOn/Group09/mappings/contratos-with-links.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+base: http://www.example.com/group09
+prefixes:
+  "": http://www.example.com/group09
+  schema: http://schema.org/
+  owl: "http://www.w3.org/2002/07/owl#"
+  db: https://es.dbpedia.org/page/
```
Comment (ocorcho) [inline HandsOn/Group09/mappings/contratos-with-links.yml:6]: No es /page/ sino /resource/

File: HandsOn/Group09/mappings/contratos-with-links.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+mappings:
+  department:
+    sources:
+      - [ContratosMenores2020-updated-with-links.csv~csv]
+    s: :/department/$(Section_IRI)
+    po:
+      - [a, org:OrganizationalUnit]
+      - [schema:name, $(Section), xsd:string]
+      - [owl:sameAs, $(wikiSection)~iri]
+#      - [org:subOrganizationOf, db:Ayuntamiento_de_Madrid]
```
Comment (ocorcho) [inline HandsOn/Group09/mappings/contratos-with-links.yml:21]: It seems to be ok.

File: HandsOn/Group09/mappings/contratos-with-links.yml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+  contractingBody:
+    sources:
+      - [ContratosMenores2020-updated-with-links.csv~csv]
+    s: :/department/$(Section_IRI)/contracting_body/$(Contracting_Body_IRI)
```
Comment (ocorcho) [inline HandsOn/Group09/mappings/contratos-with-links.yml:26]: REmove department and section IRI

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Good usage of the PPROC ontology, with only very minor errors. Links are ok as well, although we understand the difficulty of finding too many links in this context. 
You obtain a grade of 10 over 10 in this part of the evaluation

**Comment by agouloumis** [comment]
Errors solved :-)

**Pull request merged**